--- a/setup.py
+++ b/setup.py
@@ -38,7 +38,7 @@ extras_require = {
     'pycrypto': ['pycrypto >=2.6.0, <2.7.0'] + pyasn1,
     'pycryptodome': ['pycryptodome >=3.3.1, <4.0.0'] + pyasn1,
 }
-legacy_backend_requires = ['ecdsa <0.15', 'rsa'] + pyasn1
+legacy_backend_requires = ['ecdsa==0.16.0', 'rsa'] + pyasn1
 install_requires = ['six <2.0']
 
 # TODO: work this into the extras selection instead.
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 pycryptodome
 six
 rsa
-ecdsa<0.15
+ecdsa==0.16.0
 pyasn1
