--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@
 from setuptools import setup
 
 try:
-    from setuptools_rust import RustExtension
+    pass
 except ImportError:
     print(
         """
@@ -40,20 +40,7 @@
         cffi_modules=[
             "src/_cffi_src/build_openssl.py:ffi",
         ],
-        rust_extensions=[
-            RustExtension(
-                "_rust",
-                "src/rust/Cargo.toml",
-                py_limited_api=True,
-                # Enable abi3 mode if we're not using PyPy.
-                features=(
-                    []
-                    if platform.python_implementation() == "PyPy"
-                    else ["pyo3/abi3-py36"]
-                ),
-                rust_version=">=1.41.0",
-            )
-        ],
+        rust_extensions=[],
     )
 except:  # noqa: E722
     # Note: This is a bare exception that re-raises so that we don't interfere
